src/main/java/frc/team2412/robot/subsystem/IndexSubsystem.java

```diff
@@ -189,6 +194,44 @@ public boolean isFeederMotorOn() {
     public void periodic() {
         ingestBallState = ingestSensorHasBallIn();
         feederBallState = feederSensorHasBallIn();
+
+        double ingestCurrent = ingestMotor.getSupplyCurrent();
+        if (ingestCurrent > CURRENT_LIMIT_TIMEOUT3) {
+            if (ingestOverCurrentStart == 0) {
+                ingestOverCurrentStart = Timer.getFPGATimestamp();
+            }
+        }
+        if (ingestCurrent > CURRENT_LIMIT_TIMEOUT2) {
+            if (ingestOverCurrentStart > 0) {
+                if (Timer.getFPGATimestamp() - ingestOverCurrentStart > CURRENT_LIMIT_TIMEOUT1) {
+                    ingestMotorStop();
+                }
+
+            }
+        } else {
+            ingestOverCurrentStart = 0;
+
+        }
+        Timer.getFPGATimestamp();
```

Comment:
Can delete since the return value isn't used.

Reply:
done